src/packages/@ncigdc/components/FacetSelection.js

```diff
@@ -1,118 +1,110 @@
 /* @flow */
-/* eslint-disable */
 
-import React from "react";
-import _ from "lodash";
-import { css } from "glamor";
+import React from 'react';
+import _ from 'lodash';
+import { css } from 'glamor';
```

Comment:
oh wait we went the other way - so was this file missed? I don't get how files are missed when running `eslint . --fix`